src/js/wwTaskManager.js

```diff
@@ -42,9 +42,22 @@ class WwTaskManager {
     this._initEvent();
 
     this.wwe.getEditor().addEventListener('mousedown', ev => {
-      const isOnTaskBox = ev.offsetX < 18 && ev.offsetY < 18;
+      // Get position of the before element
+      const style = getComputedStyle(ev.target, ':before');
+      const left = parseInt(style.left, 10);
+      const top = parseInt(style.top, 10);
+      const width = parseInt(style.width, 10);
+      const height = parseInt(style.height, 10);
+
+      // Check if click is inside the square delimited by the before element
+      const isOnTaskBox = ev.offsetX >= left
+        && ev.offsetX <= (left + width)
+        && ev.offsetY >= top
+        && ev.offsetY <= (top + height);
```

Comment:
It's duplication code `viewer.js`'s code above.
How about extract this codes to helper function like `isOnTaskBox(style, offsetX, offsetY)` and exports this module? Then, Use this helper function in `viewr.js`

Reply:
Yeah I thought about it but I didn't known where to put it ?

Reply:
I extracted it inside `domUtils`, you can check the latest commit